src/App.css

```diff
@@ -55,3 +55,20 @@
 .nav-menu__link {
   color: #61dafb;
 }
+
+.login-status--small {
+  line-height: 1em;
+}
+
+.login-status__link {
+  color: #61dafb;
+}
+
+.login-status__link--small {
+  color: #61dafb;
```

Comment:
You don't need to repeat out a property in a modifier. You rather add the modifier class too, like `class="login-status__link login-status__link--small"`. That way the element's properties are always applied, you only need to add the changes to the modifier.